eng/Directory.Build.Data.targets

```diff
@@ -19,9 +19,26 @@
       <DirectoryPartofPath>$(PackageRootPath.Replace($(RepoRoot), ''))</DirectoryPartofPath>
       <PackageProjectUrl Condition="Exists('$(PackageRootPath)/README.md')">$([System.UriBuilder]::new($(RepositoryUrl)/blob/$(SourceRevisionId)/$(DirectoryPartofPath)README.md).Uri)</PackageProjectUrl>
       <PackageReleaseNotes Condition="Exists('$(PackageRootPath)/CHANGELOG.md')">$([System.UriBuilder]::new($(RepositoryUrl)/blob/$(SourceRevisionId)/$(DirectoryPartofPath)CHANGELOG.md).Uri)</PackageReleaseNotes>
+      <PackageReleaseNotes Condition="'$(ExtractedReleaseNotes)' != ''">$(ExtractedReleaseNotes)</PackageReleaseNotes>
     </PropertyGroup>
   </Target>
 
+  <Target Name="GetCurrentReleaseNotes" AfterTargets="Restore" BeforeTargets="Build" Condition="'$(SkipDevBuildNumber)' == 'true' and '$(IsShippingClientLibrary)' == 'true'" >
+    <PropertyGroup>
+      <PowerShellExe Condition=" '$(PowerShellExe)'=='' ">%WINDIR%\System32\WindowsPowerShell\v1.0\powershell.exe</PowerShellExe>
+      <GetReleaseNotesScriptPath Condition=" '$(GetReleaseNotesScriptPath)'=='' ">$(MSBuildThisFileDirectory)Set-PackageReleaseNotes.ps1</GetReleaseNotesScriptPath>
+      <ChangeLogPath>$([MSBuild]::NormalizeDirectory($(MSBuildProjectDirectory)/../))CHANGELOG.md</ChangeLogPath>
+      <ReleaseNotesPropsPath>$(IntermediateOutputPath)ReleaseNotes.txt</ReleaseNotesPropsPath>
```

Comment:
I thought we were going to put this into a pops file?

Reply:
Can we capture the Stdout?

Reply:
@pavelzel Yes we can capture stout, there was the possibility that logs or error messages are also included in some cases. 

@weshaggard I wasn't able to get the props file method working. I could only get the target to run at build stage and it seemed the importing of project always occurred earlier so even though I could write the props file importing it was a challenge.

Reply:
Switched to capturing console output